--- opencascade-7.4.0.orig/src/TKIVtk/EXTERNLIB
+++ opencascade-7.4.0/src/TKIVtk/EXTERNLIB
@@ -10,10 +10,3 @@
 TKService
 TKTopAlgo
 TKV3d
-vtkCommonCore
-vtkRenderingCore
-vtkRenderingOpenGL
-vtkRenderingFreeType
-vtkRenderingFreeTypeOpenGL
-vtkFiltersGeneral
-vtkInteractionStyle
+vtkCommonCore-9.0
+vtkRenderingCore-9.0
+vtkRenderingOpenGL2-9.0
+vtkRenderingFreeType-9.0
+vtkFiltersGeneral-9.0
+vtkInteractionStyle-9.0
