diff --git a/src/TKIVtk/EXTERNLIB b/src/TKIVtk/EXTERNLIB
index b8a4b250..17630fb2 100644
--- a/src/TKIVtk/EXTERNLIB
+++ b/src/TKIVtk/EXTERNLIB
@@ -10,10 +10,11 @@ TKMesh
 TKService
 TKTopAlgo
 TKV3d
-vtkCommonCore
-vtkRenderingCore
-vtkRenderingOpenGL
-vtkRenderingFreeType
-vtkRenderingFreeTypeOpenGL
-vtkFiltersGeneral
-vtkInteractionStyle
+vtkCommonCore-9.0
+vtkCommonExecutionModel-9.0
+vtkCommonDataModel-9.0
+vtkRenderingCore-9.0
+vtkRenderingOpenGL2-9.0
+vtkRenderingFreeType-9.0
+vtkFiltersGeneral-9.0
+vtkInteractionStyle-9.0
