From 5a6dab28378eeab8745bc61614901e0da9cb2f0a Mon Sep 17 00:00:00 2001
From: "Zheng, Lei" <realthunder.dev@gmail.com>
Date: Sun, 10 Oct 2021 11:03:11 +0800
Subject: [PATCH] Temporary workaround crashing when importing STEP

---
 src/StepShape/StepShape_OrientedClosedShell.cxx | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/StepShape/StepShape_OrientedClosedShell.cxx b/src/StepShape/StepShape_OrientedClosedShell.cxx
index 9625053b33..63e5bfd7cf 100644
--- a/src/StepShape/StepShape_OrientedClosedShell.cxx
+++ b/src/StepShape/StepShape_OrientedClosedShell.cxx
@@ -82,6 +82,7 @@ Handle(StepShape_Face) StepShape_OrientedClosedShell::CfsFacesValue(const Standa
 
 Standard_Integer StepShape_OrientedClosedShell::NbCfsFaces () const
 {
+  if (!closedShellElement) return 0;
   // WARNING : the field is redefined.
   // method body is not yet automaticly wrote
   // Attention, cette modif. est juste pour la compilation  
-- 
2.31.1

